4-storage/kinesis-elasticsearch-sink/src/main/scala/com.snowplowanalytics.snowplow.storage.kinesis/elasticsearch/clients/ElasticsearchSender.scala

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2016 Snowplow Analytics Ltd.
```

Comment:
2014-2016 (i.e. the copyright noticed can be thought of as global to the app's source code, not specific to the individual file)